src/Dumplie/Application/Command/Customer/AddToCart.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Command\Customer;
+
+final class AddToCart
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var int
+     */
+    private $quantity;
+
+    /**
+     * @var string
+     */
+    private $cartId;
+
+    /**
+     * AddToCart constructor.
+     * @param string $sku
+     * @param int $quantity
+     * @param string $cartId
+     */
+    public function __construct(string $sku, int $quantity, string $cartId)
+    {
+        $this->sku = $sku;
+        $this->quantity = $quantity;
+        $this->cartId = $cartId;
+    }
+
+    /**
+     * @return string
```

Comment:
There are cases where they are. For example when you expect an array of objects. I think it's easier to follow convention that we use docblocks everywhere rather than allowing for exceptions